docs/faq.md

```diff
@@ -39,26 +39,24 @@ for public repositories for free:
 * 16.0 CPUs for Arm Linux platform (Containers).
 * 8.0 CPUs for Windows platform (Containers or VMs)
 * 8.0 CPUs for FreeBSD VMs.
-* 12.0 CPUs macOS VM (1 VM).
+* 4.0 CPUs macOS VM (1 VM).
 
 Note that a single task can't request more than 8 CPUs (except macOS VMs which are not configurable).
 
-!!! note "No Monthly Minute Limit"
-    There are no limits on how many minutes a month you can use! Please keep in mind that mining cryptocurrency is against our Terms of Service, and will most likely be blocked by firewall rules and other anti-fraud mechanisms. Be a good citizen in the OSS community!
+!!! note "Monthly CPU Minutes Limit"
+    Additionally there is an upper monthly limit on free usage equal to 40 [compute credits](pricing.md)
```

Comment:
I think it's 50 compute credits here (from #1209)

Reply:
Thanks! Fixed in [cf7a6bf](https://github.com/cirruslabs/cirrus-ci-docs/pull/1208/commits/cf7a6bf4bf2c601693af689c44f6a4d4e5a730e8)